.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: node_js
 node_js:
-  - 10
```

Comment:
Why do we remove 10 though (it was an LTS an in maintenance now, at the same time 13 is EOL already)?